packages/twenty-emails/src/utils/email-renderer/marks/render-mark.tsx

```diff
@@ -0,0 +1,40 @@
+import { type JSONContent } from '@tiptap/core';
+import { type ReactNode } from 'react';
+import { type MarkType } from 'src/utils/email-renderer/email-renderer';
+import { bold } from 'src/utils/email-renderer/marks/bold';
+import { italic } from 'src/utils/email-renderer/marks/italic';
+import { link } from 'src/utils/email-renderer/marks/link';
+import { strike } from 'src/utils/email-renderer/marks/strike';
+import { underline } from 'src/utils/email-renderer/marks/underline';
+
+const MARKS_ORDER = ['underline', 'bold', 'italic', 'link', 'strike'];
```

Comment:
It's unfortunate that there is little typing. Typing should be designed so that for example you could never forget a mark that's defined on the frontend. 

Or at least it could be something like this where we don't have the same word used 3 times which would be source of error and make the document poorly readable if there are 50+ items

const MARK_RENDERERS = {
  underline, (refers to the function directly)
  bold,
  italic,
  link,
  strike,
};

const MARKS_ORDER = Object.keys(MARK_RENDERERS);